Makefile

```diff
@@ -31,10 +31,20 @@ PKG_GOLANGCI_LINT := github.com/golangci/golangci-lint/cmd/golangci-lint
 
 export GO111MODULE=on
 
-# go source files, ignore vendor directory
-SRC =  *.go cli/*.go
+# go source directories.
+# DIRS defines a single level directly, we only look at *.go in this directory.
+# REC_DIRS defines a source code tree. All go files are analyzed recursively.
+DIRS :=  ./
```

Comment:
I think this could simply be `DIRS := .` without the trailing `/`

Reply:
O.K.  Will change.